packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js

```diff
@@ -135,21 +134,14 @@ const CollectionTypeFormWrapper = ({ allLayoutData, children, slug, id, origin }
   }, [dispatch]);
 
   useEffect(() => {
-    const CancelToken = axios.CancelToken;
-    const source = CancelToken.source();
-
-    const fetchData = async (source) => {
+    const fetchData = async () => {
       dispatch(getData());
 
       try {
-        const { data } = await fetchClient.get(requestURL, { cancelToken: source.token });
+        const { data } = await fetchClient.get(requestURL);
 
         dispatch(getDataSucceeded(cleanReceivedData(cleanClonedData(data))));
       } catch (err) {
-        if (axios.isCancel(err)) {
-          return;
-        }
```

Comment:
I think these are there for a reason. react-query handles cancellation for us, but we can not just safely remove these.

Reply:
I thought it was deprecated by axios the isCancel https://axios-http.com/docs/cancellation and instead they use the signal ... my main aim was to cleanup all the instances of axios in the admin and upload package and remove the dependency from them but if we agree to just replace the axios calls to the APIs with useFetchClient it is fine to me

Reply:
It may be deprecated, but it still works. We can refactor it so use signal, but I'd be hesitant to remove these bits. It requires good testing and understanding on why they have been introduced in the first place.

Reply:
ok I understand your point, I think it is already good to have replaced all the axios call

Reply:
So whilst we have an `AbortController` in the hook, the cancelling error isn't handled (rightfully so imo) so, this we should probably re-add `axios.isCancel` or come up with a similar solution.